_posts/2018-01-02-twitter-cards-for-jekyll-with-jekyll-seo-tag.md

```diff
@@ -34,6 +34,11 @@ This will make GitHub pages know to use the plugin.
 
 To install locally run `gem install jekyll-seo-tag` at the command line.
 
+Finally, add the following tag right before `</head>` in your site's template(s):
```

Comment:
Nit, I would just say add it inside the `<head>`. It doesn't necessarily need to be right before the closing tag.